config/jobs/autobump/autobump.yaml

```diff
@@ -1,73 +1,34 @@
-periodics:
-- name: ci-prow-autobump
-  # interval: 15m
-  cron: "05 15 * * 1"  # Run at 15:05 PST (15:05 UTC) Mon
-  decorate: true
-  extra_refs:
-  # Check out the repo containing the config and deployment files for your Prow instance.
-  - org: falcosecurity
-    repo: test-infra
-    base_ref: master
-  spec:
-    containers:
-    - image: 292999226676.dkr.ecr.eu-west-1.amazonaws.com/test-infra/autobump # see images/autobump
-      imagePullPolicy: Always
-      command:
-      - /autobump.sh
-      args:
-      - /etc/github-token/oauth
-      # Make the bot name and email match the user data of the provided token's user.
-      - "Falco Automation"
-      - 51138685+poiana@users.noreply.github.com
-      - /root/gpg-signing-key/poiana.asc
-      - "EC9875C7B990D55F3B44D6E45F284448FF941C8F"
-      volumeMounts:
-      - name: service
-        mountPath: /etc/service-account
-        readOnly: true
-      - name: github
-        mountPath: /etc/github-token
-        readOnly: true
-      - name: gpg-signing-key
-        mountPath: /root/gpg-signing-key/
-        readOnly: true
-      env:
-    # autobump.sh args
-      - name: GH_PROXY
-        value: https://api.github.com
-      # GitHub org containing the repo where the Prow config and component files live.
-      - name: GH_ORG
-        value: falcosecurity
-      # GitHub repo where the Prow config and component deployment files live.
-      - name: GH_REPO
-        value: test-infra
-      # Repo relative path of the `plank` component k8s deployment file.
-      - name: PROW_CONTROLLER_MANAGER_FILE
-        value: config/prow/prow-controller-manager.yaml
-    # bump.sh args
-      # Directory containing k8s deployment YAMLs for Prow components.
-      - name: COMPONENT_FILE_DIR
-        value: config/prow/
-      # Repo relative path of the core Prow config file (config.yaml).
-      - name: CONFIG_PATH
-        value: config/config.yaml
-      # Repo relative path of the ProwJob config file or directory.
-      # Omit this if ProwJobs are only defined in config.yaml (or are not configured at all).
-      - name: JOB_CONFIG_PATH
-        value: config/jobs/
-      - name: GOOGLE_APPLICATION_CREDENTIALS
-        value: /etc/service-account/service-account.json
-    volumes:
-    - name: service
-      secret:
-        secretName: gcloud-credentials
-    - name: github
-      secret:
-        # Secret containing a GitHub user access token with `repo` scope for creating PRs.
-        secretName: oauth-token
-    - name: gpg-signing-key
-      secret:
-        secretName: poiana-gpg-signing-key
-        defaultMode: 0400
-    nodeSelector:
-      Archtype: "x86"
+periodics:
+  - cron: "05 15 * * 1"  # Run at 15:05 PST (15:05 UTC) Mon
+    name: ci-test-infra-autobump-prow
+    decorate: true
+    extra_refs:
+    - org: falcosecurity
+      repo: test-infra
+      base_ref: master
+    spec:
+      containers:
+      - image: gcr.io/k8s-prow/generic-autobumper::v20230509-5e039b951c

```

Comment:
I think there is a typo in the container image tag reference (`::`). Am I missing someting @cappellinsamuele?